torch/nn/modules/activation.py

```diff
@@ -802,9 +802,9 @@ def __init__(self, embed_dim, num_heads, dropout=0., bias=True, add_bias_kv=Fals
 
         self.add_zero_attn = add_zero_attn
 
-        self._reset_parameters()
+        self.reset_parameters()
 
-    def _reset_parameters(self):
+    def reset_parameters(self):
```

Comment:
In general, we don't want to expose those func explicitly. What's the name of `reset_parameters` in other modules? We should follow the convention there.

Reply:
Other module use reset_parameters.
for eg. 
https://github.com/pytorch/pytorch/blob/3799d1d74a39cd3d4419539453acc7f5f7c63f7e/torch/nn/modules/linear.py#L79
https://github.com/pytorch/pytorch/blob/893105b79e735bdf737394907e13a99a3b572af7/torch/csrc/api/src/nn/modules/linear.cpp#L44

Reply:
OK, then I agree the change.